_includes/navbar.html

```diff
@@ -17,7 +17,11 @@
             {% assign regions = site.data.regions | sort: "name" %}
             {% assign regions2 = regions | map: 'id' %}
             {% if regions2 contains baseurl %}
-              <span class="flag-icon flag-icon-{{ baseurl }}"></span>
+              {% if baseurl == 'ch' %}
+                <span class="flag-icon flag-icon-squared flag-icon-{{ baseurl }}"></span>
```

Comment:
I'd rather see something like this:
```suggestion
                <span class="flag-icon {% if baseurl == 'ch' %}flag-icon-squared{% endif %} flag-icon-{{ baseurl }}"></span>
```